cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -44,8 +44,12 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
         |These administrative commands are automatically routed to the `system` database when connected to the DBMS over Bolt.""".stripMargin)
     p("include::databases-command-syntax.asciidoc[]")
     section("Listing databases", "administration-databases-show-databases") {
-      p("There are three different commands for listing databases. Listing all databases, listing a particular database or listing the default database.")
-      p("All available databases can be seen using the command `SHOW DATABASES`.")
+      p(
+        """There are three different commands for listing databases. Listing all databases, listing a particular database or listing the default database.
```

Comment:
```suggestion
        """There are four different commands for listing databases. Listing all databases, listing a particular database, listing the default database, or listing the home database.
```
seems we have missed a update when adding `SHOW HOME DATABASE`

Reply:
This might need updating in earlier versions as well though